third_party/closure-compiler/Messages.properties

```diff
@@ -0,0 +1,335 @@
+#
```

Comment:
they are also part of the closure-compiler source code (just not bundled). do we need to separate this out into third_party/rhino ?